### PR TITLE
fix: 🐛 Addon.Stencil matchs nothing when use uppercase letters

### PR DESCRIPTION
**Pull request by fcten**
### Description

fix: 🐛 Addon.Stencil matchs nothing when use uppercase letters

### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Enhancement (changes that improvement of current feature or performance)
- [ ] Refactoring (changes that neither fixes a bug nor adds a feature)
- [ ] Test Case (changes that add missing tests or correct existing tests)
- [ ] Code style optimization (changes that do not affect the meaning of the code)
- [ ] Docs (changes that only update documentation)
- [ ] Chore (changes that don't modify src or test files)

### Self Check before Merge

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING**](https://github.com/antvis/x6/blob/master/CONTRIBUTING.md) document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: packages/x6/src/addon/stencil/index.ts [hunk 1]
```diff
@@ -258,7 +258,7 @@ export class Stencil extends View {
             cell,
             keyword,
             filter,
-            keyword.toLowerCase() !== keyword,
+            false,
```
Comment (NewByVector) [inline packages/x6/src/addon/stencil/index.ts:261]: 这里不需要修改吧

**Review by NewByVector — CHANGES_REQUESTED** [review]

File: packages/x6/src/addon/stencil/index.ts (hunk 1 above)
Comment (fcten) [reply packages/x6/src/addon/stencil/index.ts:261]: 如果不改的话，输入小写不会匹配到大写，输入大写可以匹配到小写，行为不一致会比较奇怪

**Review by NewByVector — CHANGES_REQUESTED** [review]

File: packages/x6/src/addon/stencil/index.ts (hunk 1 above)
Comment (fcten) [reply packages/x6/src/addon/stencil/index.ts:261]: 正常的逻辑应该是不管输入的是什么，要么总是区分大小写，要么总是不区分大小写。而原来的逻辑是输入小写的时候区分大小写，输入大写的时候不区分大小写。
所以我觉得这里固定传true或者false都ok，但是原来的判断方式很奇怪。

**Comment by NewByVector** [comment]
https://github.com/antvis/X6/pull/1211

**Pull request closed without merging**